config/providers/azure/sc-config.yaml

```diff
@@ -2,3 +2,10 @@ harbor:
   persistence:
     type: objectStorage
     disableRedirect: false
+
+fluentd:
+  logManager:
+    compaction:
+      # azcopy is used by az CLI for downloading files
+      azcopyBufferGB: 0.3
+      azcopyConcurrency: 8
```

Comment:
Could we type this out as `azureCopy` instead of `azcopy`? I think it'll make it more clear.
No one should really need to know that we are using `azcopy`, or rather that `az` cli uses `azcopy` in the background.